test/cql-pytest/cassandra_tests/validation/operations/compact_storage_test.py

```diff
@@ -1787,7 +1787,6 @@ def testGroupByWithoutPagingWithDeletions(cql, test_keyspace):
                    row(1, 2, 2, 3, 12),
                    row(1, 2, 3, 4, 12))
 
-@pytest.mark.xfail(reason="issue #5361, #5363")
```

Comment:
I see you un-xfailed some tests for #5361 and #5363 but please explain this in the commit message, and more importantly - explain if you un-xfailed **all** the tests that mentioned 5361 and 5363, or if not, why not.